nomenclature/definition.py

```diff
@@ -60,6 +62,59 @@ def validate(self, df: IamDataFrame, dimensions: list = None) -> None:
         """
         validate(self, df, dimensions=dimensions or self.dimensions)
 
+    def check_aggregate(self, df: IamDataFrame, **kwargs) -> None:
+        """Check for consistency of scenario data along the variable hierarchy
+
+        Parameters
+        ----------
+        df : :class:`pyam.IamDataFrame`
+            Scenario data to be checked for consistency along the variable hierarchy.
+        kwargs : Tolerance arguments for comparison of values
```

Comment:
I took a quick look at numpy.isclose and since there's only a total of three possible variables, maybe it would be more clear to the user to directly accept those instead of using `kwargs` here. 
Maybe `kwargs` is the way to go though as it preserves the numpy.isclose defaults. In this case perhaps listing the accepted variables in the docstring could be nice.

Reply:
I think having the link to the numpy-docs is preferable rather than repeating specific information.